### PR TITLE
add a new line before the Projects list for the six certifications in…

### LINKED ISSUE #17491: README Project links aren't rendering correctly
#### Describe your problem and - if possible - how to reproduce it

Project lists in the `README` aren't rendered correctly (see screenshot below).

#### Add a Link to the page with the problem

E.g. https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/README.md#1-responsive-web-design-certification

#### Solution

Just need to add a new line before the `Projects` list for the six certifications in the `README`.

#### If possible, add a screenshot here

image

### PR DESCRIPTION
**Pull request by mmhuntsberry**
This PR addresses issue #17491

#### Pre-Submission Checklist

- [ ] Your pull request targets the `staging` branch of freeCodeCamp.
- [ ] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [ ] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [ ] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [ ] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #17491

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,14 @@ Here are our six core certifications:
 - Responsive Web Design Principles
 - CSS Flexbox
 - CSS Grid
-** Projects **: Tribute Page, Survey Form, Product Landing Page, Technical Documentation Page, Personal Portfolio Webpage
+
+** Projects **:
```
Comment (jonathan-grah) [inline README.md]: There should be no spaces between either side of "Projects".

File: README.md [hunk 2]
```diff
@@ -46,7 +53,14 @@ Here are our six core certifications:
 - Algorithm Scripting
 - Object Oriented Programming
 - Functional Programming
-** Projects **: Palindrome Checker, Roman Numeral Converter, Caesars Cipher, Telephone Number Validator, Cash Register
+
+** Projects **:
```
Comment (jonathan-grah) [inline README.md]: There should be no spaces between either side of "Projects".

File: README.md [hunk 3]
```diff
@@ -56,27 +70,55 @@ Here are our six core certifications:
...
+  -File Metadata Microservice
 
 #### 6. Information Security and Quality Assurance Certification
 
 - Information Security with HelmetJS
 - Quality Assurance and Testing with Chai
 - Advanced Node and Express
-** Projects **: Metric-Imperial Converter, Issue Tracker, Personal Library, Stock Price Checker, Anonymous Message Board
+
+** Projects **:
```
Comment (jonathan-grah) [inline README.md]: There should be no spaces between either side of "Projects".

File: README.md [hunk 4]
```diff
@@ -56,27 +70,55 @@ Here are our six core certifications:
...
+  - Treemap Diagram
 
 #### 5. APIs and Microservices Certification
 
 - Managing Packages with Npm
 - Basic Node and Express
 - MongoDB and Mongoose
-** Projects **: Timestamp Microservice, Request Header Parser, URL Shortener, Exercise Tracker, File Metadata Microservice
+
+** Projects **:
```
Comment (jonathan-grah) [inline README.md]: There should be no spaces between either side of "Projects".

File: README.md [hunk 5]
```diff
@@ -56,27 +70,55 @@ Here are our six core certifications:
...
+  - JavaScript Calculator, 
+  - Pomodoro Clock
 
 #### 4. Data Visualization Certification
 
 - Data Visualization with D3
 - JSON APIs and Ajax
-** Projects **: Bar Chart, Scatterplot Graph, Heat Map, Choropleth Map, Treemap Diagram
+
+** Projects **:
```
Comment (jonathan-grah) [inline README.md]: There should be no spaces between either side of "Projects".

File: README.md [hunk 6]
```diff
@@ -56,27 +70,55 @@ Here are our six core certifications:
 - React
 - Redux
 - React and Redux
-** Projects **: Random Quote Machine, Markdown Previewer, Drum Machine, JavaScript Calculator, Pomodoro Clock
+
+** Projects **:
```
Comment (jonathan-grah) [inline README.md]: There should be no spaces between either side of "Projects".

**Review by jonathan-grah — CHANGES_REQUESTED** [review]
Please make sure to test your work next time before submitting a PR 👍 
Also: you should have put Closes #17491 in the description of your PR.

**Comment by mmhuntsberry** [comment]
Got it!  I'll remove and resubmit.  Thanks!

**Comment by mmhuntsberry** [comment]
I will absolutely test my work next time.  Thanks for the feedback.  I have added the Closes #17491 in the description of your PR

File: README.md [hunk 7]
```diff
@@ -34,7 +34,7 @@ Here are our six core certifications:
 - Responsive Web Design Principles
 - CSS Flexbox
 - CSS Grid
-** Projects **: Tribute Page, Survey Form, Product Landing Page, Technical Documentation Page, Personal Portfolio Webpage
+  ** Projects **: Tribute Page, Survey Form, Product Landing Page, Technical Documentation Page, Personal Portfolio Webpage
```
Comment (erictleung) [inline README.md]: These all still need the new line before them **and** the spaces should be removed around "Projects". Please review [Markdown syntax](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) if needed.

Below is more specifically what we mean by "remove spaces around 'Projects'":

```diff
- ** Projects **: Tribute Page...
+ **Projects**: Tribute Page...
```

**Review by erictleung — CHANGES_REQUESTED** [review]
Almost there. Please take a look at my comment. I hope that helps. My comment applies to all the other certificate lists as well.

File: README.md (hunk 7 above)
Comment (mmhuntsberry) [reply README.md]: Thank you for the feedback and being patient.  I removed the spaces around the 'projects' like you described (I hope) and added a new line before projects.  Hopefully this is what you mean.

**Comment by erictleung** [comment]
@raisedadead no, I was curious as to why that file was there.

**Comment by mmhuntsberry** [comment]
@raisedadead @erictleung That was my bad.  It was supposed to be a PR to a different issue.  I reverted it.  Sorry about that.  Does that clear it up?

**Comment by raisedadead** [comment]
Hi @mmhuntsberry 

That's completely alright. For now you would have to some changes that you would normally will not have to do. The best way to avoid such issues is following the [Guidelines for Contributing](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md)

Anyways follow these steps and you should be good for now (This is one time, next time just use the steps from the contributing guide)

On you local machine, within your `freeCodeCamp` dirctory:

```
git remote -v

# Output

origin	git@github.com:mmhuntsberry/freeCodeCamp.git (fetch)
origin	git@github.com:mmhuntsberry/freeCodeCamp.git (push)
upstream https://github.com/freeCodeCamp/freecodecamp.git (fetch)
upstream https://github.com/freeCodeCamp/freecodecamp.git (push)
```

If you do not have a out put like above, then follow the contributing guide to setup you local copy.
After that:

```
git checkout staging
git checkout -b fix/update-readme
git checkout staging
git reset --hard upstream/staging
git push origin staging --force
git checkout fix/update-readme
git push origin fix/update-readme
```

Come back to this pull request on GitHub `https://github.com/freeCodeCamp/freeCodeCamp/pull/17494`
Close this PR.

Open a new one from your freshly created branch.

**Comment by raisedadead** [comment]
Remember this is just one time and in future just follow the steps on the Contributing Guide. Thanks a lot for the pull request and taking the time to work on it.

**Comment by raisedadead** [comment]
You can always reachout to [Contributors Chat room](https://gitter.im/FreeCodeCamp/Contributors) for quick assistance

**Pull request closed without merging**